src/components/Form/FormLabelGroup.tsx

```diff
@@ -65,12 +67,15 @@ function FormLabelGroup({
     },
     rowClassName
   );
-  const labelClassNames = classnames({
-    'text-sm-end pe-0': !stacked,
-    'text-danger': feedback,
-    'text-success': validFeedback,
-    'visually-hidden visually-hidden-focusable': srLabel,
-  });
+  let labelClassNames = labelClassName;
+  if (!labelClassName) {
+    labelClassNames = classnames({
+      'text-sm-end pe-0': !stacked,
+      'text-danger': feedback,
+      'text-success': validFeedback,
+      'visually-hidden visually-hidden-focusable': srLabel,
+    });
+  }
```

Comment:
Nit: use coalescing operator here
```suggestion
  const labelClassNames = labelClassName ?? classnames({
    'text-sm-end pe-0': !stacked,
    'text-danger': feedback,
    'text-success': validFeedback,
    'visually-hidden visually-hidden-focusable': srLabel,
  }); 
```

Reply:
Ohhh nice! Didn't know about this, thanks